Added minimum memory allocation flag to cwl (#2862) (#5336)
DataBiosphere/toil: docs/cwl/running.rst

  --reference-inputs    Do not copy remote inputs into Toil's file store and assume they are accessible in place from all nodes.
  --disable-streaming   Do not allow streaming of job input files. By default, files marked with ``streamable`` True are streamed from remote job stores.
  --cwl-default-ram     Apply CWL specification default ramMin.
  --no-cwl-default-ram  Do not apply CWL specification default ramMin, so that Toil --defaultMemory applies. This can help jobs get to Slurm with no memory limit assigned.

>   --cwl-min-ram BYTES   Specify a minimum memory allocation for all tasks except for tools that do not specify a memory requirement and when `--no-cwl-default-ram` is passed, which is when `--defaultMemory` is used instead.

Reviewer: I don't think the top-level structure of this sentence is right; "except for (tools ((that ...) and (when ...)))" doesn't make sense because the "when" condition isn't actually qualifying a set of tools.

```suggestion
  --cwl-min-ram BYTES   Specify a minimum memory allocation for all tasks. If `--no-cwl-default-ram` is passed, this does not apply to tools that do not specify a memory requirement; `--defaultMemory` is used for those tools in that case.
```
